.github/workflows/kind_e2e_tests.yaml

```diff
@@ -92,6 +92,12 @@ jobs:
       CONTROL_PLANE: kind-k8ssandra-0
       DATA_PLANES: kind-k8ssandra-0
     steps:
+      - name: Raise open files limit
+        run: |
+          echo fs.file-max=1000000 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
```

Comment:
Maybe you can do `sudo sysctl -p` just once after the 3 echo commands?

Reply:
right 👍

Reply:
@adejanovski Not sure if you wanted to make the suggested change here, but it looks like the sysctl.conf values are still dumped 3 times here.